manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -99,13 +99,13 @@ spec:
       for: 10m
       labels:
         severity: warning
-    - alert: etcdBackendQuotaLowSpace
+    - alert: etcdDatabaseQuotaLowSpace
```

Comment:
can we separate the removal of the annotation and the vendor update into two PRs please? so we can revert them independently if necessary.

Reply:
woow, good catch, I did not mean to get these here, just removing the annotation, but the vendoring which is still not merged, got into the way .. I remove these thanks 👍🏽

Reply:
updated